util/reggen/register.py

```diff
@@ -435,6 +487,17 @@ def make_multi(self,
                     if creg_count > 1
                     else self.name)
 
+        new_alias_target = None
+        if is_alias:
+            if self.alias_target is not None:
```

Comment:
Do we need the `is_alias` argument to this function? Doesn't the constructor already set up `self.alias_target` iff `is_alias` is true?

Reply:
I believe you are right. `is_alias` would be needed if we want to keep the validation check here - but since this is already being checked in the constructor this should not be needed here. I'll remove it.